erpnext/setup/setup_wizard/operations/install_fixtures.py

```diff
@@ -68,49 +75,49 @@ def install(country=None):
 		# Stock Entry Type
 		{
 			"doctype": "Stock Entry Type",
-			"name": "Material Issue",
+			"name": _("Material Issue"),
```

Comment:
https://github.com/frappe/erpnext/blob/18bc1df6b7bbdfcf3aa526d2687eec2169b26024/erpnext/assets/doctype/asset_repair/asset_repair.py#L211-L213

Reply:
I believe it could be easier to add a `title` field and keep English `name`s?

Reply:
It does stay English, that's the point of this PR. Here, `_` is the identity function, not the translation function.

Reply:
Argh I'm sorry @barredterra, I should have read the description…